Source/ACE/Entity/Door.cs

```diff
@@ -302,7 +273,7 @@ private void Open(ObjectGuid opener = new ObjectGuid())
         private void Close(ObjectGuid closer = new ObjectGuid())
         {
             ActionChain chain = new ActionChain();
-            CurrentLandblock.ChainOnObject(chain, Guid, (WorldObject wo) =>
+            chain.AddAction(this, () =>
```

Comment:
Close is already called from an ActionChain on the door.  This chain is redundant.